Dockerfile.mayhem

```diff
@@ -0,0 +1,26 @@
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
```

Comment:
Does libkeccak need to be installed twice here? Can we copy it from the build stage?